docs/networking/vpn/tunnel-internet-traffic-with-openvpn-access-server.md

```diff
@@ -0,0 +1,80 @@
+---
+author:
+  name: Neal Sebastian
+  email: yoneal@gmail.com
+description: 'Easily tunnel web traffic through OpenVPN Access Server'
+keywords: 'openvpn,networking,vpn,tunnel'
+license: '[CC BY-ND 4.0](http://creativecommons.org/licenses/by-nd/4.0/)'
+modified: Monday, October 3rd, 2016
+modified_by:
+  name: Neal Sebastian
+  link: https://github.com/yoneal
+published: 'Friday, April 1st, 2016'
+title: 'Tunnel Internet Traffic with OpenVPN Access Server'
+---
+
+*This is a Linode Community guide. Write for us and earn $250 per published guide.*
+
+*VPN tunneling* is a method of forwarding internet traffic through your VPN connection. This is useful if you want to evade censorship, IP geolocation, or a firewall on your local network.
+
+*OpenVPN* is a popular software for creating VPN tunnels. It has two versions, *Community Edition* and *Access Server*. The Access Server is full featured and has a free license for two concurrent users. This guide details the process for configuring [OpenVPN Access Server](https://openvpn.net/index.php/access-server/overview.html) to tunnel all of your internet traffic.
+
+## Before You Begin
+
+1. Familiarize yourself with the [Getting Started](/docs/getting-started) guide.
+
+2. Follow the [Securing Your Server](/docs/security/securing-your-server) guide.  OpenVPN Access Server creates its own firewall rules, so the steps for [configuring a firewall](/docs/security/securing-your-server#configure-a-firewall) should be skipped for now. Once you've configured OpenVPN, you can apply additional firewall rules as needed.  
+
+3. Install OpenVPN Access Server using the [Secure Communications with OpenVPN Access Server](docs/networking/vpn/openvpn-access-server) guide.
+
+## Configure OpenVPN Access Server for Tunneling
+
+To configure OpenVPN for tunneling, you'll first need to log in to the Access Server Admin UI, and navigate to the **VPN Settings** page.
+
+1. In the **Routing** section, ensure that the option "Should client Internet traffic be routed through the VPN?" is set to **Yes**.
+
+    ![OpenVPN Access Server Internet Routing.](/docs/assets/openvpn-access-server-routing.png)
+
+    The option "Should VPN clients have access to private subnets (non-public networks on the server side)?" can be set to **No**, since we are using the VPN to mask internet traffic. If you wish to give VPN users access to services listening on your Linode's local network, set this option to **Yes**.
```

Comment:
What I mean is; if they want to give users access to services listening on Linode's local network, should they select **Yes, using NAT** or **Yes, using routing (advanced)**, or does that depend on additional configurations outside the scope of this guide?